packages/fxa-payments-server/src/lib/Coupon.ts

```diff
@@ -1,4 +1,8 @@
 export type Coupon = {
-  amount: number;
   promotionCode: string;
+  type: string;
+  valid: boolean;
+  discountAmount?: number;
+  expired?: boolean;
+  maximallyRedeemed?: boolean;
```

Comment:
oh... you already have a type here.  Move it to fxa-shared?  Might need to rename it so it won't be confused with Stripe's Coupon type.

Reply:
done